app/api/ada/index.js

```diff
@@ -406,33 +406,35 @@ export default class AdaApi {
       Logger.debug('AdaApi::createLedgerSignTxData called');
       const { receiver, amount } = request;
 
-      const { txExt }: AdaFeeEstimateResponse = await getAdaTransactionFee(receiver, amount);
+      const { changeAddress, txExt }: AdaFeeEstimateResponse = await getAdaTransactionFee(receiver, amount);
+
       const ledgerSignTxPayload: LedgerSignTxPayload = await createLedgerSignTxPayload(txExt);
 
       Logger.debug('AdaApi::createLedgerSignTxData success: ' + stringifyData(ledgerSignTxPayload));
       return {
         ledgerSignTxPayload,
-        // changeAddress: changeAdaAddress
+        changeAddress
       };
     } catch (error) {
       Logger.error('AdaApi::createLedgerSignTxData error: ' + stringifyError(error));
 
       if (error instanceof LocalizableError) {
         // we found it as a LocalizableError, so could throw it as it is.
         throw error;
-      } else {
-        // We don't know what the problem was so throw a generic error
-        throw new GenericApiError();
       }
+
+      // We don't know what the problem was so throw a generic error
```

Comment:
@sarveshwar-singh why did you move it outside the else?

Reply:
@nicarq 
Yes, 
at some places it's inside and some places it's outside, outside looks cleaner.

Reply:
the convention that we are following is to use if { } else { }

Reply:
@nicarq 
will revert and added issue for this PR's related non-conventional lines: https://github.com/Emurgo/yoroi-frontend/issues/308